src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -853,6 +853,21 @@
         return getCallbackRespEntityErrors(errors, caseData);
     }
 
+    private void validateCaseForCaseCloseEvent(CCDRequest ccdRequest, List<String> errors) {
```

Comment:
This and validateCaseBeforeCloseEvent in EventValidationSerivce is doing the same thing.

Reply:
No they are not doing the same thing.

Reply:
The method in the controller checks three different things(JurisdictionOutcome, HearingStatus and JudgeAllocation) where as, the one in the service checks (urisdictionOutcome, JudgementsHasJurisdiction). For the part that overlaps, the code in the service is reused.

Reply:
I created the validateCaseBeforeCloseEvent in the shared EventValidationService is so that we are running the exact same validation code for single and multiple cases on case close, preventing potential future misalignment.

Reply:
That is right. I am now adding mine on top of that. One of the previous unit tests is not happy. trying to fix that now.